test/powershell/Modules/Microsoft.PowerShell.Utility/Send-MailMessage.Tests.ps1

```diff
@@ -117,16 +166,154 @@ Describe "Send-MailMessage" -Tags CI, RequireSudoOnUnix {
         $mail = Read-Mail
 
         $mail.FromAddress | Should -BeExactly $InputObject.From
-        $mail.ToAddresses | Should -BeExactly $InputObject.To
+        $mail.ToAddresses.Count | Should -BeExactly ($InputObject.To.Count + $InputObject.Cc.Count + $InputObject.Bcc.Count)
+        $mail.ToAddresses | Should -BeIn ([array]$InputObject.To + $InputObject.Cc + $InputObject.Bcc)
 
         $mail.Headers["From"] | Should -BeExactly $InputObject.From
-        $mail.Headers["To"] | Should -BeExactly $InputObject.To
-        $mail.Headers["Reply-To"] | Should -BeExactly $InputObject.ReplyTo
-        If ($InputObject.Subject -ne $null) {
-            $mail.Headers["Subject"] | Should -BeExactly $InputObject.Subject
+        $mail.Headers["To"] | Should -Not -BeNullOrEmpty
+        $mail.Headers["To"].Split(", ") | Should -BeExactly $InputObject.To
+        If($InputObject.Cc)
+        {
+            $mail.Headers["Cc"] | Should -Not -BeNullOrEmpty
+            $mail.Headers["Cc"].Split(", ") | Should -BeExactly $InputObject.Cc
+        }
+        # BCC addresses can't be tested against mail message header, as this information is by design never included in the mail message itself.
+        If($InputObject.ReplyTo)
+        {
+            $mail.Headers["Reply-To"] | Should -BeExactly $InputObject.ReplyTo
         }
+        $mail.Headers["Subject"] | Should -BeExactly $InputObject.Subject
 
         $mail.MessageParts.Count | Should -BeExactly 1
         $mail.MessageParts[0].BodyData | Should -BeExactly $InputObject.Body
     }
 }
+
+Describe "Send-MailMessage Feature Tests" -Tags Feature, RequireSudoOnUnix {
```

Comment:
@TravisEz13 Make sense to use Feature tag?

Reply:
@iSazonov Feature tags for Pester are still available, but [feature] tag for commit messages in new PRs are now unnecessary, since all tests will be executed for PRs on the CI agents.

Reply:
I think Ilya means the `-Tags Feature` in the code

Reply:
@RDIL The way I understood it: Are `-Tags Feature` is still necessary for this kind of scenario. In my opinion it makes sense to use `Feature` tag, since the scope of the tests is much broader, than just some basic CI tests. Also those tests influence the time to run all test heavily.

Reply:
They all seem appropriate to me